packages/calcite-components/src/components/block-section/block-section.scss

```diff
@@ -62,6 +62,7 @@
 
 .toggle--switch-container {
   @apply flex items-center relative bg-transparent w-full;
+  word-break: break-word;
```

Comment:
Is there no tailwind option for this?

Reply:
nope, currently only `break-all` , `normal`  &  `keep-all` are supported